mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(saleTax) {
+  return saleTax*1.2;
```

Comment:
It looks like you've left out the space on either side of the multiplication math operator.  It makes this line of code unclear and hard to read.